package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vanilla-framework",
-  "version": "4.15.0",
+  "version": "4.15.1",
```

Comment:
Not sure when we're planning to release this PR, but I think it's close enough that we can probably ship it with 4.15.0, not 4.15.1. What exactly are our release plans @bartaz ?